.github/workflows/cmake.yml

```diff
@@ -89,6 +95,9 @@ jobs:
       # and build directories, but this is only available with CMake 3.13 and higher.  
       # The CMake binaries on the Github Actions machines are (as of this writing) 3.12
       run: |
+        # run environment setup script if any
+        [ -n "${{ matrix.config.environment_script }}" ] && "${{ matrix.config.environment_script }}"
+
         cmake $GITHUB_WORKSPACE \
           -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
           -DCMAKE_CXX_STANDARD=20 \
```

Comment:
This is useless since we are requesting it with:
```cmake
target_compile_features(cppcoro PUBLIC cxx_std_20)
```
in lib/CMakeLists.txt

Reply:
I do not understand. The environment script is about finding the compiler, setting the PATH etc. It has nothing to do with selecting the C++ version, right? Did you want to comment on another pull request?

Reply:
Noting about MSVC github action, just saying c++20 is (will be) already set within CMakeLists, I was just surprised to see it in the configuration step.

Reply:
Ah, sorry, I was thinking you are commenting on the change.

Very good question. It is really MSVC 16.8 which raises a lot of questions. It supports both C++17 and coroutines-ts as well as C++20.

What if a user has a project, which requires C++17/Coroutines-ts. Isn't it then necessary that also cppcoro is built with C++17/Coroutines-ts?
If that is the case, I'd be in favor of not setting any C++ standard, i.e. removing it from lib/CMakeLists.txt, and require the user to specify it on the command line, which implies having it in .github/workflows/cmake.yml.

What would you think?